I have no idea why this isn't included and why this wasn't an error before in 147.0.2

diff --git a/third_party/rnp/src/common/file-utils.cpp b/third_party/rnp/src/common/file-utils.cpp
index e0610f20314..90972b83ea1 100644
--- a/comm/third_party/rnp/src/common/file-utils.cpp
+++ b/comm/third_party/rnp/src/common/file-utils.cpp
@@ -35,6 +35,7 @@
 #include "uniwin.h"
 #include <errno.h>
 #else
+#include <stdlib.h>
 #include <unistd.h>
 #include <sys/types.h>
 #include <sys/stat.h>
